### ISSUE TITLE
bug: missing expected [

### ISSUE THREAD

**Issue by angela-koho** [lacework/terraform-aws-ecr#42]
**Describe the bug**
When using `lacework_ecr` module, there is an unexpected error during the terragrunt (a terraform wrapper) plan. The error message is:
```terraform
module.lacework_ecr.time_sleep.wait_time: Refreshing state... [id=2022-06-17T15:19:41Z]
╷
│ Error: missing expected [
│ 
│ 
╵
Releasing state lock. This may take a few moments...
```

**Steps to reproduce**

```terraform
terraform {
  required_version = "~> 0.15.0"

  required_providers {
    lacework = {
      source  = "lacework/lacework"
      version = "~>1.6.0" #or 1.5.1 both have the same problem
    }
  }
}
```
```terraform
module "lacework_ecr" {
  source                    = "lacework/ecr/aws"
  version                   = "~>0.7.3"
  registry_domain           = "value"
  lacework_integration_name = "ECR using IAM Role"
  use_existing_iam_role     = false
}
```

**Expected behavior**
Before upgrading API to v2, everything worked as expected. Due to API version was depreciated on Apirl 6th, 2023. I upgraded all related versions such as provider version , module version, terraform version that are required for APIv2. No code change except for version. That's why I think this error is with the module.

**Please complete the following information):**
 - Terraform Version: v0.15.0
 - Module Version : v0.7.3

**Comment by angela-koho** [comment]
Upon more testing, I noticed the issue might exist in the Lacework provider itself. 
Tried the following to validate testing:
1. Comment out pieces of the `lacework_ecr `and `lacework_ecr_iam_role` code in the terragrunt cache - the issue persisted 
2. Comment out `lacework_ecr` module completely - the issue persisted
3. Comment out lacework provider - issue resolved
This made me believe that there might be an issue existing in the Lacework Provider.

**Comment by BogdanOliynyk** [comment]
I managed to solve this issue by upgrading module `lacework_ecr` to version `0.7.2` and `lacework` provider to `1.0.1`. After first apply upgrading to latest version of module (`0.7.3`) and provider (`1.6.0`) without problems

**Comment by afiune** [comment]
@angela-koho thank you for your feedback, this issue seems to be similar or related to https://github.com/lacework/terraform-aws-ecr/issues/41 which I have tried to replicate without success. Look at comment https://github.com/lacework/terraform-aws-ecr/issues/41#issuecomment-1518290834 

I am wondering If I should use https://terragrunt.gruntwork.io/ to see if I can replicate this issue. Also, thanks to @BogdanOliynyk for posting a viable workaround.

We do acknowledge that we made a couple mistakes with our Terraform provider and this particular module, I will continue to see if we can replicate this issue so we can fix it for good. Apologies for the inconveniences.
